mailcore2-ios.podspec.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "mailcore2-ios",
+  "version": "0.6.4",
+  "summary": "Mailcore 2 for iOS",
+  "description": "MailCore 2 provide a simple and asynchronous API to work with e-mail protocols IMAP, POP and SMTP.",
+  "homepage": "http://libmailcore.com",
+  "license": {
+    "type": "BSD",
+    "file": "LICENSE"
+  },
+  "authors": "MailCore Authors",
+  "source": {
+    "git": "http://github.com/MailCore/mailcore2.git",
+    "branch": "master"
+  },
+  "platforms": {
+    "ios": 8.0
+  },
+  "header_dir": "MailCore",
+  "requires_arc": false,
+  "public_header_files": "built/include/MailCore/*.h",
```

Comment:
Could you use "cocoapods-build" as the name of the main folder for the build?
And add it to .gitignore?